### PR TITLE
Signup flow enhancements

### LINKED ISSUE #148: Replace stack.name with stack.handle throughout

### PR DESCRIPTION
**Pull request by sandersonet**
This PR fixes minor issues in the signup flow.

* Use the promise chain inside `payment-info/route.js` to update a progress bar after each promise resolves
* Add more interesting plan information to payment method form
* Refactor UI of credit card form
* Add user navigation header to top of signup so a user can log out
* Add current step indicator
* Remove current plan dropdown and instead use a small toggle at the bottom.  The idea here is that the user should have already decided on a plan.  

screenshot 2015-02-27 14 39 38

screenshot 2015-02-27 14 40 11

Also fixes #148

### CODE REVIEW

**Comment by mixonic** [comment]
I'll look at this after our sync

File: app/components/pricing-page-header/template.hbs [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{{#if isDevelopment}}
+  <h1>Add a Payment Method to Your<br /> Pay-as-you-go Development Plan</h1>
```
Comment (mixonic) [inline app/components/pricing-page-header/template.hbs]: Tiny, unimportant nit:

In HTML5, you are never required to use a `/>`. Instead, `hr` tags are part of a group of tags called `void`, meaning they can have no children element. `img` is another example.

Death to `/>`!

File: app/signup/template.hbs [hunk 2]
```diff
@@ -68,13 +70,15 @@
...
-          Run the applications and databases of your choice. Let us manage servers, network topology, security, backups, encryption,
+          We run the entire backend you need to securely host and deliver your apps: web servers, app servers, databases, load balancers, network security, backups, encryption, permissions - everything.
+        </p>
+        <p>
+          And because we run your ops, our compliance engines can help you implement an entire HIPAA compliance program at a fraction of the traditional cost.
         </p>
 
         <p>
-          Aptible will work great with your app’s language or framework.  Check out our getting started guides for help deploying your app.
+          Aptible works great with all languages and frameworks. <a href="https://support.aptible.com/quickstart" target="_blank">Check out our getting started guides</a> for help deploying your app.
```
Comment (mixonic) [inline app/signup/template.hbs]: Hm, would like to see this as:

```hbs
{{#link-to-aptible app="support" path="quickstart" target="_blank"}}
  Check out our getting started guides
{{/link-to-aptible}}
```

Thought I think `target` support for the `link-to-aptible` component would be new.

File: app/components/pricing-page-header/template.hbs (hunk 1 above)
Comment (chasballew) [inline app/components/pricing-page-header/template.hbs]: +:100: Death to `/>`

**Comment by mixonic** [comment]
Awesome.

**Pull request merged**